dashboard/test/ui/features/teacher_tools/pl_sections.feature

```diff
@@ -53,19 +55,22 @@ Feature: Professional learning Sections
     And element ".uitest-facilitator-type" is visible
     When I select facilitator participant type
 
-    # New Section details
-    Then I wait to see "#sections-set-up-container"
-    And I press keys "My Section of Teachers" for element "#uitest-section-name-setup"
-    And I wait until element "button:contains(Professional Learning)" is visible
-    And I press the first "#uitest-save-section-changes" element
-    And I wait until element "#classroom-sections" is visible
```

Comment:
I don't know cucumber syntax very well but I wonder if this `And` should be a `Then`

Reply:
I think they're actually synonyms!  https://stackoverflow.com/a/45113481